ruby/example_code/elasticbeanstalk/eb_list_stacks.rb

```diff
@@ -6,7 +6,7 @@
 # solution stacks using the AWS SDK for Ruby.
 
 # snippet-start:[eb.Ruby.listStacks]
-require "aws-sdk-elasticbeanstalk"  # v2: require 'aws-sdk'
+require "aws-sdk-elastic-beanstalk"  # v2: require 'aws-sdk'
```

Comment:
Should be `require 'aws-sdk-elasticbeanstalk'`

It is best practice to use single quote literals for strings and double ones in the case of interpolation/escaping sequences.

Reply:
Rubocop just puts double quotes on everything, regardless of usage.